noone@mail.com
Analysis of Dieting study 16S data
% Fri Sep  7 05:46:18 2018



#### \(1.2.1.2\) Taxonomic level: 4 of Subset: Patient/control samples before diet aggregated by SubjectID




##### \(1.2.1.2.1\) Loading counts and metadata

[`Subreport`](./1.2.1.2.1-report.html)


##### \(1.2.1.2.2\) Data analysis



Filtering abundance matrix with arguments [ min_mean                :10, min_quant_incidence_frac:0.25, min_quant_mean_frac     :0.25]. Filtering features


Note that some community richness estimators will not work correctly 
               if provided with abundance-filtered counts


After filtering, left 41 records for 13 features


Wrote counts and metadata for raw counts After final feature filtering to files [`data/1.2.1.2.2.1-3234ada10fbsamples.raw.16s.l.4.count.tsv`](data/1.2.1.2.2.1-3234ada10fbsamples.raw.16s.l.4.count.tsv),[`data/1.2.1.2.2.1-3234ada10fbsamples.raw.16s.l.4.attr.tsv`](data/1.2.1.2.2.1-3234ada10fbsamples.raw.16s.l.4.attr.tsv)


Wrote counts and metadata for proportions counts After final feature filtering to files [`data/1.2.1.2.2.1-3231898c8bsamples.proportions.16s.l.4.count.tsv`](data/1.2.1.2.2.1-3231898c8bsamples.proportions.16s.l.4.count.tsv),[`data/1.2.1.2.2.1-3231898c8bsamples.proportions.16s.l.4.attr.tsv`](data/1.2.1.2.2.1-3231898c8bsamples.proportions.16s.l.4.attr.tsv)



##### \(1.2.1.2.2.2\) DESeq2 tests and data normalization



Love MI, Huber W, Anders S (2014). “Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2.” _Genome Biology_, *15*,
550. doi: 10.1186/s13059-014-0550-8 (URL: http://doi.org/10.1186/s13059-014-0550-8).


\(1.2.1.2.2.2.1\) <a name="table.78"></a>[`Table 78.`](#table.78) DESeq2 results for task: Sample.type;\[ alpha:0.05\]. Full dataset is also saved in a delimited text file (click to download and open e.g. in Excel) [`data/1.2.1.2.2.2.1-3234d0c8ad6.1.2.1.2.2.2.1.a.nam.csv`](data/1.2.1.2.2.2.1-3234d0c8ad6.1.2.1.2.2.2.1.a.nam.csv)




| feature                  | baseMean | log2FoldChange | lfcSE  | stat     | pvalue    | padj    | i.baseMean | baseVar    | allZero | dispGeneEst | dispGeneIter | dispFit | dispersion | dispIter | dispOutlier | dispMAP | Intercept | Sample.type\_patient\_vs\_control | SE\_Intercept | SE\_Sample.type\_patient\_vs\_control | WaldStatistic\_Intercept | WaldStatistic\_Sample.type\_patient\_vs\_control | WaldPvalue\_Intercept | WaldPvalue\_Sample.type\_patient\_vs\_control | betaConv | betaIter | deviance | maxCooks | replace |
|:-------------------------|:---------|:---------------|:-------|:---------|:----------|:--------|:-----------|:-----------|:--------|:------------|:-------------|:--------|:-----------|:---------|:------------|:--------|:----------|:----------------------------------|:--------------|:--------------------------------------|:-------------------------|:-------------------------------------------------|:----------------------|:----------------------------------------------|:---------|:---------|:---------|:---------|:--------|
| Bacteroidales            | 6271.795 | 1.6230         | 0.4921 | 3.2985   | 0.0009721 | 0.01167 | 6271.795   | 4.289e\+07 | FALSE   | 1.1745      | 11           | 1.1745  | 1.1744     | 5        | FALSE       | 1.1744  | 11.499    | 1.6230                            | 0.3685        | 0.4921                                | 31.201                   | 3.2985                                           | 1.029e\-213           | 0.0009721                                     | TRUE     | 5        | 787.0    | NA       | FALSE   |
| Bifidobacteriales        | 12.518   | \-1.7333       | 0.6771 | \-2.5600 | 0.0104662 | 0.06280 | 12.518     | 4.443e\+02 | FALSE   | 2.3060      | 7            | 1.0568  | 2.0766     | 11       | FALSE       | 2.0766  | 4.373     | \-1.7333                          | 0.4942        | 0.6771                                | 8.848                    | \-2.5600                                         | 8.902e\-19            | 0.0104662                                     | TRUE     | 7        | 270.8    | NA       | FALSE   |
| Unclassified\_Firmicutes | 19.186   | \-0.9075       | 0.4174 | \-2.1742 | 0.0296884 | 0.09277 | 19.186     | 3.551e\+02 | FALSE   | 0.6112      | 10           | 2.9110  | 0.7736     | 9        | FALSE       | 0.7736  | 4.717     | \-0.9075                          | 0.3045        | 0.4174                                | 15.491                   | \-2.1742                                         | 4.015e\-54            | 0.0296884                                     | TRUE     | 5        | 320.8    | NA       | FALSE   |
| Erysipelotrichales       | 78.217   | 1.3032         | 0.6039 | 2.1580   | 0.0309243 | 0.09277 | 78.217     | 2.026e\+04 | FALSE   | 1.6843      | 14           | 2.4812  | 1.7514     | 11       | FALSE       | 1.7514  | 5.426     | 1.3032                            | 0.4522        | 0.6039                                | 11.998                   | 2.1580                                           | 3.632e\-33            | 0.0309243                                     | TRUE     | 6        | 426.6    | NA       | FALSE   |
| Burkholderiales          | 58.901   | 1.6637         | 0.8239 | 2.0193   | 0.0434537 | 0.10429 | 58.901     | 1.875e\+04 | FALSE   | 3.3491      | 11           | 2.6275  | 3.2672     | 11       | FALSE       | 3.2672  | 4.733     | 1.6637                            | 0.6173        | 0.8239                                | 7.667                    | 2.0193                                           | 1.759e\-14            | 0.0434537                                     | TRUE     | 13       | 362.8    | NA       | TRUE    |
| Lactobacillales          | 10.147   | 1.0224         | 0.5372 | 1.9030   | 0.0570359 | 0.11407 | 10.147     | 4.288e\+02 | FALSE   | 1.3881      | 11           | 0.6845  | 1.2747     | 11       | FALSE       | 1.2747  | 2.679     | 1.0224                            | 0.4006        | 0.5372                                | 6.686                    | 1.9030                                           | 2.294e\-11            | 0.0570359                                     | TRUE     | 6        | 270.5    | NA       | TRUE    |
| Rhizobiales              | 20.739   | \-0.9768       | 0.7821 | \-1.2489 | 0.2117185 | 0.36295 | 20.739     | 6.822e\+02 | FALSE   | 2.8083      | 11           | 3.5345  | 2.8954     | 11       | FALSE       | 2.8954  | 4.842     | \-0.9768                          | 0.5812        | 0.7821                                | 8.331                    | \-1.2489                                         | 8.013e\-17            | 0.2117185                                     | TRUE     | 10       | 309.8    | NA       | FALSE   |
| Clostridiales            | 1252.548 | \-0.3095       | 0.2734 | \-1.1322 | 0.2575704 | 0.38636 | 1252.548   | 4.524e\+05 | FALSE   | 0.3611      | 11           | 0.3640  | 0.3614     | 7        | FALSE       | 0.3614  | 10.456    | \-0.3095                          | 0.2046        | 0.2734                                | 51.108                   | \-1.1322                                         | 0.000e\+00            | 0.2575704                                     | TRUE     | 4        | 647.1    | NA       | FALSE   |
| Enterobacteriales        | 3.980    | 0.4019         | 0.8412 | 0.4778   | 0.6327921 | 0.80626 | 3.980      | 7.755e\+01 | FALSE   | 3.1366      | 12           | 0.1288  | 3.1366     | 10       | TRUE        | 1.8955  | 1.776     | 0.4019                            | 0.6224        | 0.8412                                | 2.853                    | 0.4778                                           | 4.326e\-03            | 0.6327921                                     | TRUE     | 8        | 190.9    | NA       | TRUE    |
| Selenomonadales          | 43.507   | \-0.3129       | 0.7388 | \-0.4236 | 0.6718822 | 0.80626 | 43.507     | 8.752e\+03 | FALSE   | 2.4513      | 13           | 4.8835  | 2.6173     | 10       | FALSE       | 2.6173  | 5.612     | \-0.3129                          | 0.5517        | 0.7388                                | 10.172                   | \-0.4236                                         | 2.638e\-24            | 0.6718822                                     | TRUE     | 8        | 369.3    | NA       | TRUE    |
| Unclassified\_Bacteria   | 56.023   | 0.1518         | 0.4595 | 0.3303   | 0.7411727 | 0.80855 | 56.023     | 5.693e\+03 | FALSE   | 0.8818      | 11           | 2.8216  | 1.0018     | 13       | FALSE       | 1.0018  | 5.726     | 0.1518                            | 0.3427        | 0.4595                                | 16.705                   | 0.3303                                           | 1.201e\-62            | 0.7411727                                     | TRUE     | 5        | 413.2    | NA       | FALSE   |
| Verrucomicrobiales       | 8.622    | \-0.2765       | 1.5306 | \-0.1806 | 0.8566565 | 0.85666 | 8.622      | 6.218e\+02 | FALSE   | 11.2125     | 11           | 0.4961  | 11.2125    | 12       | TRUE        | 6.5696  | 3.253     | \-0.2765                          | 1.1426        | 1.5306                                | 2.847                    | \-0.1806                                         | 4.415e\-03            | 0.8566565                                     | TRUE     | 14       | 161.5    | NA       | TRUE    |







\(1.2.1.2.2.2.1\) <a name="figure.270"></a>[`Figure 270.`](#figure.270) Clustered heatmap of normalized abundance values. Number of cluster splits is determined automatically with method `fpc::pamk`.  Image file: [`plots/3237d188edf.svg`](plots/3237d188edf.svg).
![](plots/3237d188edf.svg)




\(1.2.1.2.2.2.1\)  G-test of independence between automatic cluster splits and attribute 'Sample.type'. Number of cluster splits is determined automatically with method `fpc::pamk`.


| Test statistic | X-squared df | P value |
|:---------------|:-------------|:--------|
| 7.805          | 6            | 0.2528  |

Table: Log likelihood ratio (G-test) test of independence with Williams' correction: `m_a$attr[, main.meta.var]` and `split`


Wrote counts and metadata for raw counts Data used for heatmap with added row cluster splits to files [`data/1.2.1.2.2.2.1-32316fec49csamples.raw.htmap.count.tsv`](data/1.2.1.2.2.2.1-32316fec49csamples.raw.htmap.count.tsv),[`data/1.2.1.2.2.2.1-32316fec49csamples.raw.htmap.attr.tsv`](data/1.2.1.2.2.2.1-32316fec49csamples.raw.htmap.attr.tsv)



##### \(1.2.1.2.2.3\) Default transformations for further data analysis



Specific methods can override these and use their own normalization.


Count normalization method for data analysis (unless modified by specific methods) : [ drop.features:List of 1,  ..$ :"other", method.args  :List of 1,  ..$ theta:1, method       :"norm.ihs.prop"]


Wrote counts and metadata for raw counts Normalized after default transformations to files [`data/1.2.1.2.2.3-3237bd9898csamples.raw.16s.l.4.count.tsv`](data/1.2.1.2.2.3-3237bd9898csamples.raw.16s.l.4.count.tsv),[`data/1.2.1.2.2.3-3237bd9898csamples.raw.16s.l.4.attr.tsv`](data/1.2.1.2.2.3-3237bd9898csamples.raw.16s.l.4.attr.tsv)



##### \(1.2.1.2.3\) GeneSelector stability ranking



Slawski M, Boulesteix. A (????). _GeneSelector: Stability and Aggregation of ranked gene lists_. R package version 2.31.0.


Wilcoxon test (rank-sum for independent samples and signed-rank for paired samples) 
                   is applied to each feature (feature, gene) on random
                   subsamples of the data. Consensus ranking is found with a
                   Monte Carlo procedure ((method AggregateMC in GeneSelector package). 
                   features ordered according to the consensus ranking
                   are returned, along with the p-values, statistic and effect size 
                   computed on the full
                   original dataset. In a special case when no replications are requested,
                   features are ordered by the adjuested p-value. 
                   P-values are reported with and without the 
                   multiple testing correction of Benjamini & Hochberg. The effect sizes
                   for Wilcoxon tests are reported as: common-language effect
                   size (proportion of pairs where observations from the second group
                   are larger than observations from the first group; no effect
                   corresponds to 0.5); rank-biserial
                   correlation (common language effect size minus its complement; no
                   effect corresponds to 0; range is [-1;1]) and
                   absolute value of r (as defined in Cohen, J. (1988). Statistical power 
                   analysis for the behavioral sciences (2nd ed.). Hillsdale, NJ: Erlbaum.).
                   For paired samples, when calculating the common language effect size,
                   only paired observations are used, and one half of the number of ties is 
                   added to the numerator (Grissom, R. J., and J. J. Kim. "Effect Sizes for Research: Univariate 
                   and Multivariate Applications, 2nd Edn New York." NY: Taylor and Francis (2012)).
                   Logarithm in base 2 of the fold change (l2fc) is also reported if requested.
                   For independent samples, the fold change is computed between the sample means of
                   the groups (last to first). For paired samples - as the sample median of the logfold change
                   in each matched pair of observations.


Stability selection parameters are: [ comp.log.fold.change:TRUE, maxrank             :20, samp.fold.ratio     :0.5, replicates          :400, type                :"unpaired", block.attr          : NULL]


\(1.2.1.2.3\)  Summary of response variable (unpaired samples) Sample.type.



```````
control patient 
     18      23 
```````



\(1.2.1.2.3\) <a name="table.79"></a>[`Table 79.`](#table.79) GeneSelector stability ranking for response Sample.type.  When fold change or difference is computed, this is done as 'patient by control'. Full dataset is also saved in a delimited text file (click to download and open e.g. in Excel) [`data/1.2.1.2.3-323e985a14.1.2.1.2.3.a.name.ta.csv`](data/1.2.1.2.3-323e985a14.1.2.1.2.3.a.name.ta.csv)




| name                     | statistic | pval      | pval.adjusted | common.lang.eff.size | rank.biserial.corr.eff.size | r.eff.size | l2fc.patient.by.control | mean.control | mean.patient |
|:-------------------------|:----------|:----------|:--------------|:---------------------|:----------------------------|:-----------|:------------------------|:-------------|:-------------|
| Clostridiales            | 69.0      | 0.0001458 | 0.001620      | 0.1667               | \-0.66667                   | 0.59316    | \-1.123260              | 0.336638     | 0.154535     |
| Unclassified\_Firmicutes | 74.0      | 0.0002700 | 0.001620      | 0.1787               | \-0.64251                   | 0.56887    | \-1.807642              | 0.008502     | 0.002429     |
| Bacteroidales            | 332.0     | 0.0007154 | 0.002862      | 0.8019               | 0.60386                     | 0.52843    | 0.486003                | 0.482778     | 0.676160     |
| Unclassified\_Bacteria   | 91.0      | 0.0017435 | 0.005231      | 0.2198               | \-0.56039                   | 0.48894    | \-0.746321              | 0.014395     | 0.008581     |
| Rhizobiales              | 106.0     | 0.0063310 | 0.015194      | 0.2560               | \-0.48792                   | 0.42637    | \-1.734093              | 0.006004     | 0.001805     |
| Bifidobacteriales        | 114.0     | 0.0125706 | 0.025141      | 0.2754               | \-0.44928                   | 0.38976    | \-3.066250              | 0.014998     | 0.001791     |
| Selenomonadales          | 158.0     | 0.2002966 | 0.343366      | 0.3816               | \-0.23671                   | 0.20001    | \-1.915717              | 0.023169     | 0.006141     |
| Enterobacteriales        | 165.0     | 0.2549778 | 0.382467      | 0.3986               | \-0.20290                   | 0.17778    | \-0.003909              | 0.017678     | 0.017630     |
| Erysipelotrichales       | 247.0     | 0.3028988 | 0.403865      | 0.5966               | 0.19324                     | 0.16090    | 1.357161                | 0.010161     | 0.026031     |
| Lactobacillales          | 184.5     | 0.5583928 | 0.670071      | 0.4457               | \-0.10870                   | 0.09140    | \-0.525545              | 0.019265     | 0.013384     |
| Burkholderiales          | 198.5     | 0.8246013 | 0.824601      | 0.4795               | \-0.04106                   | 0.03461    | \-0.389999              | 0.011891     | 0.009075     |
| Verrucomicrobiales       | 198.5     | 0.8052285 | 0.824601      | 0.4795               | \-0.04106                   | 0.03851    | 1.609551                | 0.003447     | 0.010518     |





##### \(1.2.1.2.4\) Stability selection analysis for response ( Sample.type )




##### \(1.2.1.2.5\) PermANOVA (adonis) analysis of  normalized counts



Oksanen J, Blanchet FG, Friendly M, Kindt R, Legendre P, McGlinn D, Minchin PR, O'Hara RB, Simpson GL, Solymos P, Stevens MHH, Szoecs E, Wagner
H (2018). _vegan: Community Ecology Package_. R package version 2.5-2, <URL: https://CRAN.R-project.org/package=vegan>.


Non-parametric multivariate test for association between
                           normalized counts and meta-data variables. Dissimilarity index is euclidean.


\(1.2.1.2.5\)  Association with the patient/control status unpaired with formula count\~Sample.type .



```````
Permutation test for adonis under reduced model
Terms added sequentially (first to last)
Permutation: free
Number of permutations: 4000

adonis2(formula = as.formula(formula_str), data = m_a$attr, permutations = perm, method = dist.metr)
            Df SumOfSqs      R2      F    Pr(>F)    
Sample.type  1   0.7215 0.18705 8.9737 0.0004999 ***
Residual    39   3.1358 0.81295                     
Total       40   3.8573 1.00000                     
---
Signif. codes:  0 ‘***’ 0.001 ‘**’ 0.01 ‘*’ 0.05 ‘.’ 0.1 ‘ ’ 1
```````






\(1.2.1.2.5\)  Association with the patient/control status unpaired Adonis summary.


|     &nbsp;      | Df | SumOfSqs | R2     | F     | Pr(>F) |
|:---------------:|:---|:---------|:-------|:------|:-------|
| **Sample.type** | 1  | 0.7215   | 0.1871 | 8.974 | 5e-04  |
|  **Residual**   | 39 | 3.136    | 0.8129 | NA    | NA     |
|    **Total**    | 40 | 3.857    | 1      | NA    | NA     |

Table: Permutation test for adonis under reduced model


\(1.2.1.2.5\)  Association with the age quartiles and patient/control status with formula count\~age.quant \* Sample.type .



```````
Permutation test for adonis under reduced model
Terms added sequentially (first to last)
Permutation: free
Number of permutations: 4000

adonis2(formula = as.formula(formula_str), data = m_a$attr, permutations = perm, method = dist.metr)
                      Df SumOfSqs      R2       F    Pr(>F)    
age.quant              3   0.1535 0.03979  0.7324 0.5876031    
Sample.type            1   0.7562 0.19603 10.8264 0.0007498 ***
age.quant:Sample.type  3   0.6428 0.16664  3.0678 0.0269933 *  
Residual              33   2.3049 0.59754                      
Total                 40   3.8573 1.00000                      
---
Signif. codes:  0 ‘***’ 0.001 ‘**’ 0.01 ‘*’ 0.05 ‘.’ 0.1 ‘ ’ 1
```````






\(1.2.1.2.5\)  Association with the age quartiles and patient/control status Adonis summary.


|          &nbsp;           | Df | SumOfSqs | R2     | F      | Pr(>F) |
|:-------------------------:|:---|:---------|:-------|:-------|:-------|
|       **age.quant**       | 3  | 0.1535   | 0.0398 | 0.7324 | 0.5876 |
|      **Sample.type**      | 1  | 0.7562   | 0.196  | 10.83  | 7e-04  |
| **age.quant:Sample.type** | 3  | 0.6428   | 0.1666 | 3.068  | 0.027  |
|       **Residual**        | 33 | 2.305    | 0.5975 | NA     | NA     |
|         **Total**         | 40 | 3.857    | 1      | NA     | NA     |

Table: Permutation test for adonis under reduced model


Count normalization method for abundance plots : [ drop.features:List of 1,  ..$ :"other", method.args  : list(), method       :"norm.prop"]



##### \(1.2.1.2.6\) Plots of Abundance.

[`Subreport`](./1.2.1.2.6-report.html)




\(1.2.1.2.6\) <a name="figure.297"></a>[`Figure 297.`](#figure.297) Clustered heatmap of normalized abundance values. Number of cluster splits is determined automatically with method `fpc::pamk`.  Image file: [`plots/3237067f083.svg`](plots/3237067f083.svg).
![](plots/3237067f083.svg)




\(1.2.1.2.6\)  G-test of independence between automatic cluster splits and attribute 'Sample.type'. Number of cluster splits is determined automatically with method `fpc::pamk`.


| Test statistic | X-squared df | P value   |
|:---------------|:-------------|:----------|
| 5.196          | 1            | 0.0226  * |

Table: Log likelihood ratio (G-test) test of independence with Williams' correction: `m_a$attr[, main.meta.var]` and `split`


person) (????). _morpheus: Interactive heat maps using 'morpheus.js' and 'htmlwidgets'_. R package version 0.1.1.1, <URL:
https://github.com/cmap/morpheus.R>.





\(1.2.1.2.6\) <a name="widget.87"></a>[`Widget 87.`](#widget.87) Dynamic Morpheus heatmap of normalized abundance values. 
                    It is available here through the link only because it can take a while to render for large datasets.
                    This is very customizable. What you will see initially is just a default starting configuration. Explore its menus. Click to see HTML widget file in full window: [`./1.2.1.2.6-32352acc9f4Dynamic.Morpheus.hea.html`](./1.2.1.2.6-32352acc9f4Dynamic.Morpheus.hea.html)


Wrote counts and metadata for raw counts Data used for heatmap with added row cluster splits (clustering by abundance profile) to files [`data/1.2.1.2.6-32316513122samples.raw.htmap.count.tsv`](data/1.2.1.2.6-32316513122samples.raw.htmap.count.tsv),[`data/1.2.1.2.6-32316513122samples.raw.htmap.attr.tsv`](data/1.2.1.2.6-32316513122samples.raw.htmap.attr.tsv)



##### \(1.2.1.2.7\) Ordinations, 

[`Subreport`](./1.2.1.2.7-report.html)


##### \(1.2.1.2.8\) Network Analysis Feature correlation with overlaid differential abundance results 

[`Subreport`](./1.2.1.2.8-report.html)